adyenofficial.php

```diff
@@ -1646,7 +1684,11 @@ public function hookPayment()
             return null;
         }
 
-        $this->context->controller->addCSS('modules/' . $this->name . '/views/css/adyen.css', 'all');
+        $loadAdyenCss = Configuration::get('ADYEN_ENABLE_CHECKOUT_STYLING');
+
+        if ($loadAdyenCss) {
+            $this->context->controller->addCSS('modules/' . $this->name . '/views/css/adyen.css', 'all');
```

Comment:
is this going to disable the checkout component css as well or only the ones that we added as extra for prestashop adjustments?

Reply:
Nope. The file which will not be imported only contains css specific for the prestashop plugin